karate-tests/src/test/java/patients/TestParallel.java

```diff
@@ -16,7 +16,7 @@ void testDevParallel() {
                 .outputJunitXml(true)
                 .tags("~@sandbox-only", "~@smoke-only")
                 .karateEnv("veit07")
-                .parallel(5);
+                .parallel(3);
```

Comment:
Am I right in understanding this as approximately, it runs 3 processes from which it sends these test requests?

Reply:
yeah 3 threads will be used to run the tests in parallel, it allows karate to execute multiple test scenarios simultaneously

Reply:
Was there a motivation to change from 5 to 3?

Reply:
tests are using single p9 user(9900000285) across multiple scenarios and started getting version mismatch errors(while one test is making get request for patient another test might have made the patch for the same and version number been updated).I have changed the parallel count as tests are not taking much time

Reply:
👍